### PR TITLE
Additional accounts can now be created easily

### PR DESCRIPTION
**Pull request by annando**
We can now create additional accounts really easy.

### CODE REVIEW

File: src/Module/Register.php [hunk 1]
```diff
@@ -43,12 +43,20 @@ public static function content(array $parameters = [])
...
+		if (local_user() && $block) {
 			notice('Permission denied.' . EOL);
 			return '';
 		}
 
-		if ((!local_user()) && (intval(Config::get('config', 'register_policy')) === self::CLOSED)) {
+		if (local_user()) {
+			$user = DBA::selectFirst('user', ['parent-uid'], ['uid' => local_user()]);
+			if (!empty($user['parent-uid'])) {
+				notice('Only parent users can create additional profiles.');
```
Comment (tobiasd) [inline src/Module/Register.php]: The notice should be translate-able.

File: src/Module/Register.php [hunk 2]
```diff
@@ -43,12 +43,20 @@ public static function content(array $parameters = [])
...
+				return '';
+			}
+		}
+
+		if (!local_user() && (intval(Config::get('config', 'register_policy')) === self::CLOSED)) {
 			notice('Permission denied.' . EOL);
```
Comment (tobiasd) [inline src/Module/Register.php]: This notice as well (and further below, the string is already translate-able)

**Comment by tobiasd** [comment]
It does create a new delegation right?

**Comment by annando** [comment]
It creates that new user and the delegation for it.

**Comment by tobiasd** [comment]
Associated discussion in the [support forum](https://forum.friendi.ca/display/ec054ce7-165e-2160-8432-11a736799491).

File: src/Module/Register.php [hunk 3]
```diff
@@ -43,13 +43,21 @@ public static function content(array $parameters = [])
...
+			notice(L10n::t('Permission denied.'));
 			return '';
 		}
 
-		if ((!local_user()) && (intval(Config::get('config', 'register_policy')) === self::CLOSED)) {
-			notice('Permission denied.' . EOL);
+		if (local_user()) {
+			$user = DBA::selectFirst('user', ['parent-uid'], ['uid' => local_user()]);
+			if (!empty($user['parent-uid'])) {
+				notice(L10n::t('Only parent users can create additional profiles.'));
```
[the remark this answers is not in the document]
Comment (annando) [reply src/Module/Register.php]: Done

**Comment by AlfredSK** [comment]
Can a user create e.g. forums even when registration is closed?

**Comment by AlfredSK** [comment]
Hmm. Where is it? I don't see any difference besides the changed headline.

**Comment by annando** [comment]
What do you see under `/settings/delegation`?

**Comment by AlfredSK** [comment]
The ability to configure a parent user and delegates. That's it. Just like before. :-)

**Comment by annando** [comment]
Ah yeah. I see what I did wrong. You will see that link after you registered your first additional account.

**Pull request merged**